guides/micronaut-azure-http-functions/micronaut-azure-http-functions.adoc

```diff
@@ -84,28 +84,34 @@ You can invoke the GET route:
 [source,bash]
 ----
 curl -i localhost:7071/api/micronautguide
+----
+
+```
 HTTP/1.1 200 OK
 Date: Sat, 08 May 2021 03:55:11 GMT
 Content-Type: text/plain
 Server: Kestrel
 Transfer-Encoding: chunked
 
 Example Response
-----
+```
 
 or the POST route:
 
 [source,bash]
 ----
 curl -i -d '{"name":"John Snow"}' -H "Content-Type: application/json" -X POST http://localhost:7071/api/micronautguide
+----
+
+```
```

Comment:
I am not sure that this is proper asciidoc. I think they are Github fenced code blocks